src/main/java/com/yahoo/sketches/quantiles/FlexDoublesArrayAccessor.java

```diff
@@ -0,0 +1,56 @@
+package com.yahoo.sketches.quantiles;
+
+import java.util.Arrays;
+
+public class FlexDoublesArrayAccessor extends DoublesBufferAccessor {
```

Comment:
what does the name represent?
need some explanation on why a special accessor is needed